### PR TITLE
implement http lib

### PR DESCRIPTION
**Pull request by AlbertHambardzumyan**

### CODE REVIEW

File: src/lib/http/http.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+   */
+  constructor () {
+    this.signKey = null
+    this.clientId = null
+  }
+
+  /**
+   * @param {string} signKey
+   * @param {string} clientId
+   * @description Set config
```
Comment (manushak) [inline src/lib/http/http.js]: forget `. ` at the end

File: src/lib/http/http.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  }
+}
+
+Http.HOST = 'https://api.renderforest.com'
+Http.DEFAULT_OPTIONS = {
+  method: 'GET',
+  json: true,
+  headers: {
+    'Accept': 'application/json',
+    'User-agent': `renderforest/sdk-node/${PackageJson.version}`
```
Comment (manushak) [inline src/lib/http/http.js]: what do you think 'User-Agent' could be better?

File: src/lib/http/http.js (hunk 1 above)
Comment (narekhovhannisyan) [reply src/lib/http/http.js]: :D

File: src/lib/http/http.js (hunk 2 above)
Comment (AlbertHambardzumyan) [reply src/lib/http/http.js]: Got shout

**Pull request merged**